front/cypress/e2e/native_survey_page_action.cy.ts

```diff
@@ -122,12 +122,24 @@ describe('Native survey project page actions', () => {
     cy.get('[type="checkbox"]').check();
     cy.setLoginCookie(userEmail, userPassword);
     cy.visit(`/projects/${projectSlugTimeline}`);
-    cy.contains('Take the survey').click({ force: true });
+    cy.get('[data-testid="e2e-project-survey-button"]').click({ force: true });
     cy.contains('New submissions can only be added in active phases.').should(
       'exist'
     );
   });
 
+  it('tests actions when unregistered users may submit survey responses', () => {
+    cy.setAdminLoginCookie();
+    cy.visit(`admin/projects/${projectIdContinuous}/permissions`);
+    cy.get('#e2e-granular-permissions').within(() => {
+      cy.contains('Everyone').click({ force: true });
+    });
```

Comment:
The Radio component in the component library doesn't seem to accept data-testid or data-cy attributes, and the id is a generated one. We should update the Radio component to accept these attributes, but for now I've used a "contains" within the context of the granular permissions wrapper.